content/dstu2/care-plan.md

```diff
@@ -0,0 +1,50 @@
+---
+title: CarePlan | DSTU 2 API
+---
+
+# CarePlan
+
+* TOC
+{:toc}
+
+## Terminology Bindings
+
+<%= terminology_table(:care_plan, :dstu2) %>
+
+## Search
+
+Search for CarePlans that meet supplied query parameters:
+
+    GET /CarePlan?:parameters
+
+### Parameters
+
+ Name         | Required?                                                                                                        | Type          | Description
+--------------|------------------------------------------------------------------------------------------------------------------|---------------|-------------------------------------------------------------------------------------
+ `_id`        | This or `patient`/`subject`. If populated: `date`, `patient`, and `subject` are not allowed; `_count` is ignored | [`token`]     | The logical resource id associated with the resource.
```

Comment:
Re-wrote the table using Notes
https://github.com/cerner/fhir.cerner.com/pull/105/commits/5ad8fd719446a3188c9888d6371fff5860e52b40#diff-ccd172a35d4a08ccbd7f98c90c538e79L22